main.nf

```diff
@@ -890,16 +895,20 @@ process megahit {
 process spadeshybrid {
     tag "$id"
     publishDir "${params.outdir}/", mode: 'copy', pattern: "${id}*",
-        saveAs: {filename -> filename.indexOf(".fastq.gz") == -1 ? "Assembly/SPAdesHybrid/$filename" : null}
+        saveAs: {filename -> 
+          if (filename.indexOf(".fastq.gz") == -1  && filename.indexOf("_scaffolds.fasta") == -1) "Assembly/SPAdesHybrid/$filename"
+          else if (filename.indexOf("_scaffolds.fasta.gz") > 0) "Assembly/SPAdesHybrid/$filename"
+          else null}
```

Comment:
same as above

Reply:
Probably not the same, but
```
saveAs: {filename -> 
          if (filename.indexOf(".log") > 0 || filename.indexOf("_scaffolds.fasta.gz") > 0 || filename.indexOf("_graph.gfa.gz") > 0 || filename.indexOf("_contigs.fasta.gz") > 0 ) "Assembly/$filename"
          else null}
```
would be better as well.

Reply:
I changed that.